### ISSUE TITLE
Are sub-types supported ? 

### ISSUE THREAD

**Issue by kohsah** [cirsfid-unibo/lime#26]
I noticed in structure.json, the `docTypes`structure takes only the high-level docType name ("doc", "bill" ... ). Is it possible to have two "doc" docTypes but having a different sub-Type name ? something like  : 

```

  "docTypes": [
      {
         "name": "doc",
         "subType": "adoc",
          locales: [ ... ]
     },
     {
         "name": "doc",
         "subType": "bdoc",
          locales: [ ... ]
    }
]
```

**Comment by obujor** [comment]
By now the subtype is taken from the document on marking of the "docType" element, and it is shown only in the uri.

**Comment by kohsah** [comment]
Hi @obujor I am working on supporting languagePlugins by documentType + subType . I have multiple documents of type debateReport which differ on subType .. and a similar case with "doc" document Type.

**Comment by obujor** [comment]
Great! Do the subType documents need different buttons which are not included in documentType? Where you would like to insert the subType in Akoma?

**Comment by kohsah** [comment]
Yeah there are cases where they would need different buttons - not different buttons per se... but for 1 type I don't want to show some buttons . I have a case where the two documents are very similar in structure (modeled on debateReport) and other cases where the documents are structurally different (even if modeled on doc). I think this is pretty much a valid scenario when you consider the generic doc / debateReport types.

So in the current scenario when I say new document, it is a AN document type that is provided, but in reality you could have multiple document types in an organization mapped to a AN document type ... and you differentiate based on sub-type.

**Comment by obujor** [comment]
Ok, this makes sense.
Thanks!

**Comment by kohsah** [comment]
@obujor  I am testing this functionality at the moment, will be sending a PR in a day or two.

**Comment by obujor** [comment]
The pull request #36 provides an implementation for this feature but we were waiting for a rework to merge it, finally we closed the pull request because the rework wasn't submitted.
